src/app/api/updateTask/[id]/route.ts

```diff
@@ -0,0 +1,39 @@
+import prisma from "@/lib/prisma";
+import { z } from "zod";
+
+const ZTaskSchema = z.object({
+  id: z.number().optional(),
+  title: z.string(),
```

Comment:
**Nit: z.string().min(1)** to make sure teh strings are non empty

Reply:
done